constructor/osx/update_path.sh

```diff
@@ -22,13 +22,25 @@ BASH_RC_BAK="${BASH_RC}-__NAME_LOWER__.bak"
 cp -fp $BASH_RC ${BASH_RC_BAK}
 
 echo "
-Prepending PATH=$PREFIX/bin to PATH in $BASH_RC
+Initializing __NAME__ in $BASH_RC
 
-For this change to become active, you have to open a new terminal.
-"
-echo "
+For this change to become active, you have to open a new terminal."
+
+cat <<EOF >> "$BASH_RC"
 # added by __NAME__ __VERSION__ installer
-export PATH=\"$PREFIX/bin:\$PATH\"" >>$BASH_RC
+# >>> conda initialize >>>
+# !! Contents within this block are managed by 'conda init' !!
+if [ -f "$PREFIX/etc/profile.d/conda.sh" ]; then
+    . "$PREFIX/etc/profile.d/conda.sh"
+else
+    export PATH="$PREFIX/bin:\$PATH"
+# <<< conda initialize <<<
+EOF
```

Comment:
Same as above.  Use what `conda init` will be putting in I think.

Reply:
Same as before. Don't want users to blame Anaconda for making their shell instantiation slow.